lib/modules/manager/bazel-module/extract.spec.ts

```diff
@@ -93,6 +93,39 @@ describe('modules/manager/bazel-module/extract', () => {
       );
     });
 
+    it('returns bazel_dep with no version and git_override', async () => {
+      const input = codeBlock`
+        bazel_dep(name = "rules_foo")
+        git_override(
+            module_name = "rules_foo",
+            commit = "850cb49c8649e463b80ef7984e7c744279746170",
+            remote = "https://github.com/example/rules_foo.git",
+        )
+        `;
+      const result = await extractPackageFile(input, 'MODULE.bazel');
+      if (!result) {
+        throw new Error('Expected a result.');
+      }
+      expect(result.deps).toHaveLength(2);
+      expect(result.deps).toEqual(
```

Comment:
use single `toMatchObject`
```suggestion
      expect(result).toMatchObject(
```